tests/cacl/test_cacl_application.py

```diff
@@ -17,6 +17,18 @@
 ]
 
 
+@pytest.fixture(scope="module", autouse=True)
+def disable_port_toggle(duthosts, tbinfo):
+    # set mux mode to manual on both TORs to avoid port state change during test
+    if "dualtor" in tbinfo['topo']['name']:
```

Comment:
this check will alow if the testbed is a active-active dualtor: `dualtor-aa`.

Reply:
an alternative is using fixture `active_standby_ports` and check if the returned list is empty. 

https://github.com/sonic-net/sonic-mgmt/blob/9903dfa0be9d6b8c5d522dc8ed0682ed478ae2de/tests/common/dualtor/dual_tor_common.py#L85